themes/jaeger-docs/layouts/partials/navbar.html

```diff
@@ -23,7 +24,7 @@
       </div>
     </div>
 
-    <div id="topNav" class="navbar-menu">
+    <div id="topNav" class="navbar-menu is-active">
```

Comment:
I think this one is flipping menu to active on resize

Reply:
Whoops! Ugh, I keep doing this. Fixed.